### PR TITLE
Slosh wallet updates

### PR DESCRIPTION
**Pull request by MrSlosh**
Removed mining UI and code. Fixed the minimum transaction fee to 0 but added a "dust" fee to prevent users from dust spamming the network.
Also updated graphic elements to conform to logo that jamesburrell2 requested to be used.

### CODE REVIEW

File: src/main.h [hunk 1]
```diff
@@ -33,7 +33,8 @@ static const unsigned int MAX_BLOCK_SIZE = 1000000;
 static const unsigned int MAX_BLOCK_SIZE_GEN = MAX_BLOCK_SIZE/2;
 static const unsigned int MAX_BLOCK_SIGOPS = MAX_BLOCK_SIZE/50;
 static const unsigned int MAX_ORPHAN_TRANSACTIONS = MAX_BLOCK_SIZE/100;
-static const int64 MIN_TX_FEE = 100000000;
```
Comment (cmelion) [inline src/main.h]: This means the minimum transaction fee used to be 1 NYC (if I calculated that right)?

File: src/main.h [hunk 2]
```diff
@@ -1572,7 +1573,7 @@ class CAlert : public CUnsignedAlert
     bool CheckSignature()
     {
         CKey key;
-        if (!key.SetPubKey(ParseHex("040184710fa689ad5023690c80f3a49c8f13f8d45b8c857fbcbc8bc4a8e4d3eb4b10f4d4604fa08dce601aaf0f470216fe1b51850b4acf21b179c45070ac7b03a9")))
+        if (!key.SetPubKey(ParseHex("04AFC6B9D279C647C67C250BC4C68F5E9BD714867D8D45B6E54FEEFFA6CB9074EE7CBF69EB8FF388304604C2E00661601502B42E3CF71E934B4A5EAB0D5B209BFD")))
             return error("CAlert::CheckSignature() : SetPubKey failed");
```
Comment (cmelion) [inline src/main.h]: Where does this public key come from?  Explanation of use?

File: src/net.cpp [hunk 3]
```diff
@@ -1144,7 +1144,14 @@ void MapPort()
...
-       {"107.155.190.222", "107.155.190.222"}
+       {"24.190.179.138", "24.190.179.138"},
+       {"69.118.35.147", "69.118.35.147"},
+       {"5.101.122.184", "5.101.122.184"},
+       {"107.155.190.222", "107.155.190.222"},
+       {"24.141.187.231", "24.141.187.231"},
+       {"50.82.94.170", "50.82.94.170"},
+       {"82.33.54.166", "82.33.54.166"},
+       {"118.208.191.227", "118.208.191.227"}
 };
```
Comment (cmelion) [inline src/net.cpp]: Nice!

File: src/main.h (hunk 1 above)
Comment (MrSlosh) [reply src/main.h]: Correct.

File: src/main.h (hunk 2 above)
Comment (MrSlosh) [reply src/main.h]: I believe that is Hobbyists public key used to silence the alert system. This should be removed in the next pull from his code. I will confirm.

File: src/net.cpp (hunk 3 above)
Comment (MrSlosh) [reply src/net.cpp]: Indeed. I believe the nodes will be removed in a future update in favor of seeded DNS

File: src/main.h (hunk 2 above)
Comment (cmelion) [reply src/main.h]: I think we are good for now, the community is used to ignoring the alerts already.  And we have enough of a social media presence to address the unlikely event it is misused.

**Comment by cmelion** [comment]
```
test/Checkpoints_tests.cpp(24): error in "sanity": check !Checkpoints::CheckBlock(1500, p120000) failed
test/Checkpoints_tests.cpp(25): error in "sanity": check !Checkpoints::CheckBlock(120000, p1500) failed
test/DoS_tests.cpp(128): error in "DoS_checknbits": check !CheckNBits(firstcheck.second, lastcheck.first+60*60*24*14, lastcheck.second, lastcheck.first) failed
test/key_tests.cpp(59): error in "key_test1": check bsecret1.SetString (strSecret1) failed
test/key_tests.cpp(60): error in "key_test1": check bsecret2.SetString (strSecret2) failed
test/key_tests.cpp(61): error in "key_test1": check bsecret1C.SetString(strSecret1C) failed
test/key_tests.cpp(62): error in "key_test1": check bsecret2C.SetString(strSecret2C) failed
test/key_tests.cpp(84): error in "key_test1": check addr1.Get() == CTxDestination(key1.GetPubKey().GetID()) failed
test/key_tests.cpp(85): error in "key_test1": check addr2.Get() == CTxDestination(key2.GetPubKey().GetID()) failed
test/key_tests.cpp(86): error in "key_test1": check addr1C.Get() == CTxDestination(key1C.GetPubKey().GetID()) failed
test/key_tests.cpp(87): error in "key_test1": check addr2C.Get() == CTxDestination(key2C.GetPubKey().GetID()) failed

*** 11 failures detected in test suite "Bitcoin Test Suite"
```

**Comment by Hobbyistpool** [comment]
A quick look at test/Checkpoints_tests.cpp(24), it appears that this hash (841a2965955dd288cfa707a755d05a54e45f8bd476835ec9af4402a2b59a2967)  is still litecoin.  The tests were never updated for NYC.  I assume the others would be the same,

**Comment by MrSlosh** [comment]
Tests modified to reflect data actually on the NYC blockchain. Only one test failed due to deficiencies in the DoS codebase.

**Pull request merged**